app/views/admin/users/show.html.haml

```diff
@@ -14,12 +14,22 @@
           - if can? :edit, @user
             .pull-right
               = link_to 'Edit', edit_admin_user_path(@user), class: 'btn btn-primary'
-          - if @user.confirmed?
-            %span.label.label-success
-              confirmed
+          %b
```

Comment:
This can now be part of the table, and the items can then be properly aligned with everything else!